api/src/main/resources/workbench-api.yaml

```diff
@@ -2339,6 +2339,21 @@ paths:
           schema:
             type: boolean
             default: false
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/cohorts/{cohortId}/extractGenomesJob":
```

Comment:
I would drop the `Job` part and just make it a verb; my general preference is just to use a verb for non-standard rest methods (and POST, which you already did), per https://cloud.google.com/apis/design/custom_methods (it also suggests using a `:`, but we don't really use that pattern and it might make Swagger unhappy)